funtofem/interface/fun3d_interface.py

```diff
@@ -258,8 +258,23 @@ def set_functions(self, scenario, bodies):
 
         for function in scenario.functions:
             if function.adjoint:
-                start = 1 if function.stop == -1 else function.start
-                stop = 1 if function.stop == -1 else function.stop
+                timing_not_defined = function.stop is None or function.start is None
```

Comment:
Can we treat the stop and start being none separately? It seems this implementation would always set start = 1 **and** stop = 1 (or steps) even if only one is undefined.